.env.example

```diff
@@ -1,2 +1,4 @@
 VUE_APP_INFURA_KEY=
-VUE_APP_IS_MAINNET=false
\ No newline at end of file
+VUE_APP_OFFLINE_DB=tknbrgoff
+VUE_APP_SIDE_CHAIN_ID=1 # If not set, uses 1 as default
```

Comment:
Missing MAIN_CHAIN_ID variable, also make sure to make checks for the main/side combinations